lib/rouge/lexers/make.rb

```diff
@@ -73,7 +73,7 @@ def initialize(opts={})
       end
 
       state :export do
-        rule %r/[\w[\$]{1,2}{}()-]/, Name::Variable
+        rule %r/[\w\${}()-]/, Name::Variable
```

Comment:
Not really this PR's fault, but we usually ask for `+` at the end of single-char-class regexes like this for perf reasons. This one's old enough it might have been my fault 😅

Reply:
I'm fine adding the `+` ... but:

do you see how the `{1,2}` is also inside of a class, so it isn't a length specifier at all?... you might want to change the regexp to match more correctly.